src/asset.js

```diff
@@ -160,6 +160,21 @@ export class Asset {
     return null;
   }
 
+  /**
+   * @returns {xdr.AssetType}  the raw XDR representation of the asset type
+   */
+  getRawAssetType() {
```

Comment:
💭  I don't think it should block this PR, but thoughts on keeping `getAssetType` and `getRawAssetType` to use the same conditional logic? Imo this `getRawAssetType` logic makes more sense, so that null isn't a possible return val (my understanding is there shouldn't ever be an asset that has a code > 12 or 0). Not sure what the standard is for changing legacy code, but thought worth mentioning

Reply:
Good idea! Done in 473595b.